bundles/framework/admin-announcements/Tile.js

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Doesn't really look like you need a custom tile. What does it do differently than the default tile?

Reply:
removed tiles and used extended classes in flyout and instance